doc/howtos/rdtraining/setup.rst

```diff
@@ -0,0 +1,408 @@
+
+
+Configure odoo-dev
+------------------
+
+Odoo-dev is the internal R&D repository, it's in this repository that you will create your working
+branches. This repository is only available if you are working at Odoo.
+
+.. code-block:: console
+
+    $ cd /home/$USER/src/odoo
+    $ git remote add odoo-dev git@github.com:odoo-dev/odoo.git #add odoo-dev as remote
+    $ git remote rename origin odoo #change the name of origin (the odoo repository) to odoo
+    $ git remote set-url --push odoo no_push #remove the posibilities to push to odoo (you can only push to odoo-dev)
+
+    $ cd /home/$USER/src/enterprise
+    $ git remote add enterprise-dev git@github.com:odoo-dev/enterprise.git
+    $ git remote rename origin enterprise
+    $ git remote set-url --push enterprise no_push
+
+
+Useful git commands
+-------------------
+
+These are useful git commands for your every working day.
+
+* Change branch
+  When you change branch, both repositories (odoo and enterprise) must be synchronized, i.e., both
+  need to be in the same branch.
+
+  .. code-block:: console
+
+    $ cd /home/$USER/src/odoo
+    $ git checkout saas-13.5
+
+    $ cd /home/$USER/src/enterprise
+    $ git checkout saas-13.5
+
+* Fetch and rebase :
+
+  .. code-block:: console
+
+    $ cd /home/$USER/src/odoo
+    $ git fetch --all
+    $ git rebase --autostash odoo/saas-13.5
+
+    $ cd /home/$USER/src/enterprise
+    $ git fetch --all
+    $ git rebase --autostash enterprise/saas-13.5
+
+
+Install the dependencies
+========================
+
+Python
+------
+
+Odoo requires Python 3.6 or later, if your computer is updated you should already have an updated version.
+
+You can check your Python version with:
+
+.. code-block:: console
+
+    $ python3 --version
+
+Install pip3 and libraries
+--------------------------
+
+For libraries using native code, it is necessary to install development tools and native
+dependencies before the Python dependencies of Odoo.
+
+.. code-block:: console
+
+    $ sudo apt install python3-pip
+    $ sudo apt install python3-dev libxml2-dev libxslt1-dev libldap2-dev libsasl2-dev
+    $ sudo apt install libssl-dev libpq-dev libjpeg-dev
+
+
+Install odoo requirements
+-------------------------
+
+.. code-block:: console
+
+    $ cd /home/$USER/src/odoo
+    $ pip3 install -r requirements.txt
+
+
+Install wkhtmltopdf
+-------------------
+
+wkhtmltopdf is a library to render HTML into PDF, it is used to create the PDF reports. wkhtmltopdf
+is not installed through pip and must be installed manually in version 0.12.5 for it to support
+headers and footers.
+
+.. code-block:: console
+
+    $ cd /tmp/
+    $ sudo wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.focal_amd64.deb
+    $ sudo gdebi --n wkhtmltox_0.12.5-1.focal_amd64.deb
+    $ sudo ln -s /usr/local/bin/wkhtmltopdf /usr/bin
+    $ sudo ln -s /usr/local/bin/wkhtmltoimage /usr/bin
```

Comment:
so there is no issue with roboto in Linux Mint 20?

Reply:
It seems to work correctly, the PDF is generated and it has its headers and footers.